lib/engine/game/base.rb

```diff
@@ -1436,6 +1436,14 @@ def round_description(name, round_number = nil)
         description.strip
       end
 
+      def can_select?(_entity)
+        true
+      end
+
+      def companies_in_bank
```

Comment:
this isn't necessary, you can just do game.companies.select { ... }

Reply:
Ok.